integration/docker/entrypoint.sh

```diff
@@ -52,6 +52,7 @@ case ${service,,} in
     integration/docker/bin/alluxio-master.sh
     ;;
   worker)
+    export ALLUXIO_RAM_FOLDER=${ALLUXIO_RAM_FOLDER:-/dev/shm}
```

Comment:
@aaudiber Will the behavior be affected because we set `ALLUXIO_RAM_FOLDER` earlier and it will be picked up into site properties? I think that is actually intended behavior right?